tests/sentry/options/test_store.py

```diff
@@ -27,12 +32,24 @@ def make_key(self, ttl=10, grace=10):
 
     def test_simple(self):
         store, key = self.store, self.key
+
         assert store.get(key) is None
         assert store.set(key, 'bar')
         assert store.get(key) == 'bar'
         assert store.delete(key)
+
+    def test_simple_without_cache(self):
```

Comment:
Should add a test here that actually returns a value, this is just testing the failures.